scapy/route6.py

```diff
@@ -360,3 +360,6 @@ def route(self, dst="", dev=None, verbose=conf.verb):
 
 
 conf.route6 = Route6()
+
+# Reload interfaces
+conf.ifaces.reload()
```

Comment:
That's a bit crappy. We only need to reload the default interface, not the entire interface list